SR2021/2020-02-28-srawn-03.md

```diff
@@ -0,0 +1,39 @@
+The Student Robotics (Almost) Weekly Newsletter is a (roughly) fortnightly newsletter which summarises the goings‐on across Student Robotics. **Emboldened** items are ones for which additional help has been requested.
+
+# Headlines
+
+### Mentoring
+
+**There are a number of teams who need mentors to help them with this year's game.** No experience with mentoring or the simulator required, but experience with SR is definitely useful!
+
+If you're interested, please let the competition team know!
+
+### Module III League
```

Comment:
Would be super awesome to use the Roman numeral here (and elsewhere) when referring to the modules:
```suggestion
### Module Ⅲ League
```
The league sessions however are numbered using Arabic numerals.

Reply:
TIL that these are included in unicode!